afs-storage-api/src/test/java/com/powsybl/afs/storage/AbstractAppStorageTest.java

```diff
@@ -538,7 +538,7 @@ public void test() throws IOException, InterruptedException {
         assertNotNull(storage.getEventsBus());
 
         storage.flush();
-        eventStack.clear();
+        Thread.sleep(200);
```

Comment:
Is it really OK to sleep like this?

Reply:
yea, the test is flaky in client server mode on windows, this sleep seems to fix it

Reply:
For me sleep is not ok at all. We have to understand what is the real issue. I guess it is related to the changes of this PR as we never encountered this kind of error with the way we have to test events (synchro is supposed to be handled)

Reply:
i reproduced this error in another PR, so this is not due to changes here..

Reply:
also I couldn't reproduce the error on a windows vm..

Reply:
If we remove the sleep, which assertion fails? Is it always the same?

Reply:
yes, this is fixed properly now